src/Roslyn.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22415.0
+VisualStudioVersion = 14.0.22512.0
```

Comment:
you can run ```git reset HEAD Roslyn.sln```

git docs are [here](http://schacon.github.io/git/git-reset.html)

I believe the pending changes window in Visual Studio will also allow you to unstage a commit.